src/osx/cocoa/nonownedwnd.mm

```diff
@@ -335,8 +335,20 @@ - (id) init
     return self;
 }
 
-- (BOOL) triggerMenu:(SEL) action
+- (BOOL) triggerMenu:(SEL) action sender:(id)sender
```

Comment:
Sorry, I don't understand what do you mean: this is from your commit c5394cea63ef184f59da26ae4feb4f55c9d9bf01, did you change your mind since then? But, just to keep things simple: should this commit be merged or reverted as well?

Reply:
Sorry, I tried to put notes to individual commits, but apparently I failed. Just wanted to comment that this commit still looks fine to me.